src/main/java/org/apache/sling/distribution/agent/impl/SimpleDistributionAgentQueueProcessor.java

```diff
@@ -173,6 +176,16 @@ private boolean processQueueItem(String queueName, DistributionQueueEntry queueE
         return removeItemFromQueue;
     }
 
+    private void addRandomDelay(){
+        int random = (int)(30 * Math.random() + 0);
+        try {
+            Thread.sleep(random * 1000);
+        } catch (InterruptedException e) {
```

Comment:
nit: conventionally, ignored execeptions should be named differently
```suggestion
        } catch (InterruptedException ign) {
```